tools/cloud-build/images/test-runner/Dockerfile

```diff
@@ -29,13 +29,21 @@ RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add -  && \
     dnsutils && \
     # install terraform and packer
     apt-add-repository "deb [arch=$(dpkg --print-architecture)] https://apt.releases.hashicorp.com bullseye main" && \
-    apt-get -y update && apt-get install -y unzip python3-pip python3-netaddr terraform packer jq && \
+    apt-get -y update && apt-get install -y unzip python3-pip python3-netaddr terraform packer jq apt-transport-https && \
     # install gcloud
     echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" \
       | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && \
     curl https://packages.cloud.google.com/apt/doc/apt-key.gpg \
       | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
     apt-get -y update && apt-get -y install google-cloud-sdk && \
+    mkdir -p /etc/apt/keyrings && \
```

Comment:
Can we do `gcloud components install kubectl`  instead ?

https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#install_kubectl

Reply:
Let me try that, I tried doing something similar but that failed so let me get the error messages again.